src/Apps/Order2/Routes/Checkout/Components/Order2CheckoutLoadingSkeleton.tsx

```diff
@@ -33,6 +33,7 @@ export const Order2CheckoutLoadingSkeleton: React.FC<
               orderData.itemsTotal?.display) as string
           }
         />
+        <Order2ExpressCheckoutSkeleton />
```

Comment:
Talked to Vivi and we don't need a loading skeleton for express checkout - it will just be included in the general lines of `StepsSkeleton` below.